doc/tech/usecases.md

```diff
@@ -11,3 +11,12 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+Content 
+•	Objective ( High level Objective ) 
```

Comment:
Can you please use markdown syntax (- or *) for bulleted list?

Reply:
sure I will  :)